src/components/pages/article/article-page.test.tsx

```diff
@@ -1,11 +1,55 @@
-import { render } from '@testing-library/react';
+import { render, screen, cleanup } from '@testing-library/react';
 import { ArticlePage } from './article-page';
 import {
   content, metaData, peerReview, status,
 } from '../../../utils/mocks';
+import { ArticleFiguresTab, ArticleFullTextTab, ArticleReviewsTab } from './tabs';
 
 describe('ArticlePage', () => {
   it('renders correctly', () => {
-    expect(() => render(<ArticlePage metaData={metaData} content={content} status={status} peerReview={peerReview}/>)).not.toThrow();
+    expect(() => render(<ArticlePage metaData={metaData} status={status} activeTab="fulltext">
+        <ArticleFullTextTab content={content} peerReview={peerReview} metaData={metaData} />
+      </ArticlePage>)).not.toThrow();
+  });
+
+  it('renders with figures tab', () => {
+    expect(() => render(<ArticlePage metaData={metaData} status={status} activeTab="figures">
+        <ArticleFiguresTab content={content} />
+      </ArticlePage>)).not.toThrow();
+  });
+
+  it('renders with figures tab', () => {
```

Comment:
is this supposed to say `renders with reviews tab`?

Reply:
What would you say if I said no? :P

I've pushed the correction